src/css/style.css

```diff
@@ -72,4 +73,115 @@
 .map-icon {
 	position: relative;
 	top: 2px;
+}
+
+footer {
```

Comment:
Please use CSS class selector instead of tag selector, you might wanna read about the BEM convention.